micro_dl/cli/image_inference.py

```diff
@@ -94,7 +94,15 @@ def parse_args():
     return args
 
 
-def run_prediction(args, gpu_ids, gpu_mem_frac):
+def run_prediction(model_dir,
```

Comment:
I'm so sorry, I moved this file to deprecated in the previous PR. We can keep this in cli, let's remove it from deprecated!

Reply:
How about we keep it in cli until I've tested the inference module with data other than 3D and updated the notebook, then I can move it to deprecated.

Reply:
sorry what I meant was move it from deprecated to cli!